src/templates/en/2020/stories/user_experience.html

```diff
@@ -0,0 +1,3918 @@
+              <span data-amp-original-style="color: #fcfcfc" class="amp-wp-4a61a81">HTTP Archive's annual</span>
+              <span data-amp-original-style="font-weight: 900; color: #fcfcfc"
+                    class="amp-wp-f94bb8b">state of the web</span><span data-amp-original-style="color: #fcfcfc"
+                                                                        class="amp-wp-4a61a81"> report</span></p></div>
+          </div>
+          <div
+            data-amp-original-style="position:absolute;pointer-events:none;left:-5.58252%;top:57.76699%;width:134.70874%;height:41.26214%;transform:rotate(354deg);opacity:1"
+            class="amp-wp-10b0a45">
+            <div class="mask amp-wp-a1b70b8" id="el-90c94fbb-06eb-4862-b819-0f046d5ab371"
+                 data-amp-original-style="pointer-events:initial;width:100%;height:100%;display:block;position:absolute;top:0;left:0">
+              <div class="fill amp-wp-a511151" data-amp-original-style="background-color:#5b667a"></div>
+            </div>
+          </div>
+          <div
+            data-amp-original-style="position:absolute;pointer-events:none;left:-8.25243%;top:62.45955%;width:134.70874%;height:41.26214%;transform:rotate(354deg);opacity:1"
+            class="amp-wp-15b7dc0">
+            <div class="mask amp-wp-a1b70b8" id="el-d9fba4a5-b10e-4b24-9f82-2f3992c2bf23"
+                 data-amp-original-style="pointer-events:initial;width:100%;height:100%;display:block;position:absolute;top:0;left:0">
+              <div class="fill amp-wp-b3b6893" data-amp-original-style="background-color:#545f74"></div>
+            </div>
+          </div>
+          <div
+            data-amp-original-style="position:absolute;pointer-events:none;left:9.95146%;top:79.61165%;width:80.09709%;height:43.85113%;opacity:1"
+            class="amp-wp-e8247a0">
+            <div class="mask amp-wp-a1b70b8" id="el-a2577370-d911-435c-a21d-826228c08231"
+                 data-amp-original-style="pointer-events:initial;width:100%;height:100%;display:block;position:absolute;top:0;left:0">
+              <div data-leaf-element="true"
+                   data-amp-original-style="position:absolute;width:100%;height:100.08762%;left:0%;top:-0.04381%"
+                   class="amp-wp-3fc8e8f">
+                <amp-img layout="fill" src="/static/stories/web-almanac-2020-user-experience/WA_Chat-Window.png" alt="A graphic showing an example website"
+                         srcset="/static/stories/web-almanac-2020-user-experience/WA_Chat-Window.png 994w, /static/stories/web-almanac-2020-user-experience//static/stories/web-almanac-2020-user-experience/WA_Chat-Window-768x631.png 768w, /static/stories/web-almanac-2020-user-experience//static/stories/web-almanac-2020-user-experience/WA_Chat-Window-300x247.png 300w, /static/stories/web-almanac-2020-user-experience//static/stories/web-almanac-2020-user-experience/WA_Chat-Window-150x123.png 150w"
```

Comment:
Lots and lots of these. Copy and replace error?

```suggestion
                         srcset="/static/stories/web-almanac-2020-user-experience/WA_Chat-Window.png 994w, /static/stories/web-almanac-2020-user-experience/WA_Chat-Window-768x631.png 768w, /static/stories/web-almanac-2020-user-experience/WA_Chat-Window-300x247.png 300w, /static/stories/web-almanac-2020-user-experience/WA_Chat-Window-150x123.png 150w"
```

Reply:
Yep :) My MacBook's keyboard is a bit rusty, so it accidentally did CMD+V twice, hence everything is doubled.